Zend/tests/attributes/002_rfcexample.phpt

```diff
@@ -0,0 +1,41 @@
+--TEST--
+Attributes: RFC Example
+--FILE--
+<?php
+namespace My\Attributes {
+    use PhpAttribute;
+
+    <<PhpAttribute>>
+    class SingleArgument {
+        public $argumentValue;
+
+        public function __construct($argumentValue) {
+             $this->argumentValue = $argumentValue;
+        }
+    }
+}
+
+namespace {
+    use My\Attributes\SingleArgument;
+
+    <<SingleArgument("Hello World")>>
```

Comment:
This kind of usage, combined with the scenario `Attributes can be placed on all supported elements.`, seems very fragile/hard to use.

Specifically, I can already see the most common error scenario to be somebody asking "why didn't I get the string that I was expecting" due to a missing import.

Very powerful, and I don't mind (I think code style rules can be used to enforce attributes to reference existing symbols), but a bit error-prone.